chapters/02-Datenstruktur.Rmd

```diff
@@ -115,18 +115,18 @@ Das geht auch in verschachtelten Listen:
 # Definiere eine Liste, die eine Liste und einen Vektor enthält:
 # (Anm.: hier habe ich keine Namen für die Elemente vergeben)
 
-mylist <- list(list(1,"a"),  
+mylist <- list(list(1, "a"),  
                vec_4)        
 
-# Rufe Element 1 der äußeren Liste: (1,"a"), und davon dann Element 2 ab:
+# Rufe Element 1 der äußeren Liste: (1, "a"), und davon dann Element 2 ab:
 
 mylist[[1]][2]                        
 ```
 Ich habe jetzt mehrere Variablen (Values, Vektoren, Listen) definiert, sie sind in meinem RStudio im `Environment`-tab oben rechts aufgetaucht.
 
 ## Der Workspace
 
-Rechts oben im Fenster ist das `Environment`-tab. Hier sieht man alle im `globale Workspace` definierten Objekte (Datenstrukturen: Werte, Vektoren, Matrizen, Arrays, Listen, `data.frames`, `tibbles`; und Funktionen) aufgelistet:
+Rechts oben im Fenster ist das `Environment`-tab. Hier sieht man alle im `global Workspace` definierten Objekte (Datenstrukturen: Werte, Vektoren, Matrizen, Arrays, Listen, `data.frames`, `tibbles`; und Funktionen) aufgelistet:
```

Comment:
Should we use a different term (i.e., not "tab")? I'm being picky here but some might get confused because it not what they would see as a "tab".